### PR TITLE
Da: fk20 encoder

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Integrate fk20 in encoder
Fixed evaluation domains sizes and usage

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
Test fail is not related

File: nomos-da/kzgrs-backend/src/dispersal.rs [hunk 1]
```diff
@@ -250,6 +250,7 @@ mod tests {
         verifiers: &[DaVerifier],
     ) -> Vec<Attestation> {
         let mut attestations = Vec::new();
+        let domain_size = encoded_data.extended_data.0[0].len();
```
Comment (megonen) [inline nomos-da/kzgrs-backend/src/dispersal.rs:253]: I assumed this gives us the row size (for extended data)

**Review by megonen — APPROVED** [review]
👍

File: nomos-da/kzgrs-backend/src/dispersal.rs (hunk 1 above)
Comment (danielSanchezQ) [reply nomos-da/kzgrs-backend/src/dispersal.rs:253]: Yes, just for the test.

**Pull request merged**